kolibri/utils/options.py

```diff
@@ -352,3 +352,24 @@ def update_options_file(section, key, value, KOLIBRI_HOME, ini_filename="options
             file=conf.filename
         )
     )
+
+
+def generate_empy_options_file(options_path, options_data):
+
+    exclude_key = ["PICKLE_PROTOCOL"]
+
+    # Generate an options.ini file inside the KOLIBRI_HOME as default placeholder config
+    file = open(options_path, "w")
+    for k, v in options_data.items():
+        file.write("# [{}] \n".format(k))
+        loop_count = 0
+        for k_child, v_child in v.items():
+            loop_count += 1
+            if k_child not in exclude_key:
```

Comment:
since we are going to add a newline character to the end of each block, I'm wondering if we can change to the following for simplicity:
```
if k_child not in exclude_key:
    file.write("# {} = {} \n".format(k_child, v_child))
if loop_count == len(v):
    file.write("\n")
```